scripts/doc_checker.py

```diff
@@ -483,6 +479,18 @@ def monkeypatch(*args, **kwargs):
         return lambda cls_or_func: cls_or_func
 
     ray.remote = monkeypatch
+    modin.utils.instancer = lambda cls: cls
+
+    # monkey patch numpydoc for correct work with properties
+    old_load_obj = Docstring._load_obj
+
+    def load_obj(name):
```

Comment:
```suggestion
    # monkey-patch numpydoc for working correctly with properties
    def load_obj(name, old_load_obj=Docstring._load_obj):
```
slightly more compact :)